activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1724,6 +1735,12 @@ def check_constraint_for!(table_name, expression: nil, **options)
             raise(ArgumentError, "Table '#{table_name}' has no check constraint for #{expression || options}")
         end
 
+        def assert_valid_deferrable(deferrable)
```

Comment:
Let's move this method to activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb